diff --git a/modules/contrib/context_mobile_condition/src/Plugin/Condition/MobileDetectCondition.php b/modules/contrib/context_mobile_condition/src/Plugin/Condition/MobileDetectCondition.php
index 14a0bad..dc7e5ab 100644
--- a/modules/contrib/context_mobile_condition/src/Plugin/Condition/MobileDetectCondition.php
+++ b/modules/contrib/context_mobile_condition/src/Plugin/Condition/MobileDetectCondition.php
@@ -115,7 +115,8 @@ public function buildConfigurationForm(array $form,
         $configuration = $this->getConfiguration();
         $form['mobile_detect_condition'] = [
         '#title' => $this->t('Mobile detect'),
-        '#type' => 'radios',
+        // '#type' => 'radios',
+        '#type' => 'checkbox',
         '#options' => array(
         '0' => $this->t('Mobile Device'),
         '1' => $this->t('Tablet Device'),
